portfolio/src/main/webapp/script-home.js

```diff
@@ -68,3 +69,21 @@ function createListElement(text) {
   liElement.innerText = text;
   return liElement;
 }
+
+/*
+ * Creates a map using the Google Maps API.
+ */
+function initMap() {
+  const script = document.createElement('script');
+  script.src = 'https://maps.googleapis.com/maps/api/js?key=AIzaSyBI_pU_plN3tOYv6h0SohPZ7qWBWafJgvs&callback=createMap';
+  script.defer = true;
+  script.async = true;
+  // The callback function is attached to the 'window'
+  window.createMap = function() {
```

Comment:
Please write comment mentioning the name of this callback "createMap" is supplied in the map scripts src URL. 

I imagine this is how Google communicates with ur application, which is why it has to go in the window object. Otherwise, this is fishy lol

Reply:
You didnt mention in the comment that the callback is supplied to the API via the URL params. Also change the order of ur code. 
Create the callback first, then create the mapScript. Make it abundantly clear that the callback is supplied as a url parameter.

Reply:
Holy hell, is this really how it works? You arbitrarily add a function to the window, then encode the name of that function in the URL for the source of 

That looks so sketchy lol